### PR TITLE
Add formatting script to Attack Pattern Indicator

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Status
- [x] Ready

## Related Issues
related: demisto/etc#37594
related PR - #13222

## Description
Add formatting script to Attack Pattern Indicator.

## Does it break backward compatibility? 
- [x] No

## Must have
- [x] Tests

### CODE REVIEW

File: Packs/CommonScripts/Scripts/ExtractAttackPattern/ExtractAttackPattern.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  default: true
+  isArray: true
+scripttarget: 0
+subtype: python3
+runonce: false
+dockerimage: demisto/python3:3.9.6.22912
+runas: DBotWeakRole
+tests:
+- No test
+fromversion: 5.0.0
```
Comment (yuvalbenshalom) [inline Packs/CommonScripts/Scripts/ExtractAttackPattern/ExtractAttackPattern.yml:23]: why not 5.5.0?

**Review by yuvalbenshalom — COMMENTED** [review]
see my comments

File: Packs/CommonScripts/Scripts/ExtractAttackPattern/ExtractAttackPattern.yml (hunk 1 above)
Comment (ChanochShayner) [reply Packs/CommonScripts/Scripts/ExtractAttackPattern/ExtractAttackPattern.yml]: done

**Comment by ChanochShayner** [comment]
@yaron-libman Can you please go over the release notes? Thanks. (Please do not push the changes you require)

**Comment by ShirleyDenkberg** [comment]
@ChanochShayner Sent updated files to you by Slack.

**Review by yuvalbenshalom — APPROVED** [review]

**Pull request merged**